### PR TITLE
Unable default drag/pinch rotate on map interactions (Add this intera…

### PR DESCRIPTION
**Pull request by edellucien**
…ction for north arrow example).

### CODE REVIEW

File: src/components/BasicMap/BasicMap.js [hunk 1]
```diff
@@ -132,11 +136,34 @@ class BasicMap extends Component {
       zoom,
     } = this.props;
 
+    this.dfltInteractions = defaultInteractions({
```
Comment (oterral) [inline src/components/BasicMap/BasicMap.js]:
```suggestion
    const dfltInteractions = defaultInteractions({
```

File: src/components/BasicMap/BasicMap.js [hunk 2]
```diff
@@ -132,11 +136,34 @@ class BasicMap extends Component {
...
+        map
+          .getInteractions()
+          .getArray()
+          .forEach(i => {
+            if (i instanceof DragRotate || i instanceof PinchRotate) {
+              map.removeInteraction(i);
+            }
+          });
+      }
+    }
```
Comment (oterral) [inline src/components/BasicMap/BasicMap.js]: I don't think you need this block . If user provide a Map with interactions it's up to him to manage his interactions.

File: src/components/BasicMap/BasicMap.js [hunk 3]
```diff
@@ -132,11 +136,34 @@ class BasicMap extends Component {
...
+
     this.map =
       map ||
       new OLMap({
         controls: [],
-        interactions: interactions || defaultInteractions(),
+        interactions: interactions || this.dfltInteractions,
```
Comment (oterral) [inline src/components/BasicMap/BasicMap.js]: Put the defaultinteractiond efinition here:

```
efaultInteractions({
      altShiftDragRotate: false,
      pinchRotate: false,
    });
```

File: src/components/BasicMap/BasicMap.js (hunk 2 above)
Comment (edellucien) [reply src/components/BasicMap/BasicMap.js]: My requirement for this task is to deactivate rotation,pinch rotate, etc by default in react-spatial.

Comment (edellucien) [reply src/components/BasicMap/BasicMap.js]: This requirement will not be fulfilled, if I don;t unset by default rotation. But still allow the user to add interactions directly in props if needed.

Comment (edellucien) [reply src/components/BasicMap/BasicMap.js]: quote from mario: 'Wenn keine Map übergeben wird, initialisieren wir die Karte ohne Rotation.'
so I removed it :+1:

**Pull request merged**